docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -72,13 +74,11 @@ Adding jitter to the base delay value adds a layer of chaotic unreliability that
 
 Adding packet loss, apart from introducing even more effective delay to our system could also wreak havoc on our unreliable messages, thus allowing us to explore if we need more defensive logic surrounding our unreliable messages or if we should opt for a reliable message instead.
 
-Testing Boss Room on desktop, we've used artificial lag between 80ms and 120ms with 5%-10% packet loss.
-
 ### Different network conditions for different peers
 
 :::important
 
-[Clumsy](#clumsy-on-windows) and [Network Link Conditioner](#network-link-conditioner-on-osx) are introducing changes on OS level, thus all the instances of the game that we open on our local machine would run under the same network conditions. 
+[Clumsy](#clumsy-on-windows), [Network Link Conditioner](#network-link-conditioner-mac-os) and [dummynet](#dummynet-dnctl-and-pftcl-mac-os) are introducing changes on OS level, thus all the instances of the game that we open on our local machine would run under the same network conditions. 
```

Comment:
not just games, all programs including your web browsers. It might be worth it point out to not forget to disable it once you're done, else your computer will feel slow :P

Reply:
agreed, added!